Symfony/CS/Tests/Fixer/Symfony/PhpdocScalarFixerTest.php

```diff
@@ -256,4 +256,28 @@ public function testEmptyDocBlock()
 
         $this->makeTest($expected);
     }
+
+    public function testPhpdocHasAWrongTagNameCase()
```

Comment:
as well as the second one, and this test name doesn't tell us what we are testing, the second test is better and may be the only one added test.